src/main/java/org/example/Main.java

```diff
@@ -335,18 +425,105 @@ public static MutableGraph removeEdge(MutableGraph g, String srcLabel, String ds
         return g;
     }
 
-    public static String graphSearch(MutableGraph g, Node src, Node dst, type t) {
-        String result = "";
-        if (t == type.BFS)
+    public static String graphSearch(MutableGraph g, MutableNode src, MutableNode dst, Algorithm t) {
+        String result;
+        if (t == Algorithm.BFS)
         {
-            BFS(g, src, dst, t);
+            System.out.println("Conducting BFS");
+            result = DoBFS(g, src, dst);
         }
         else
         {
-            DFS(g, src, dst, t);
+            System.out.println("Conducting DFS");
+            result = DoDFS(g, src, dst);
         }
         return result;
     }
+    public static void walkSearch(MutableGraph g, MutableNode src, MutableNode dst)
```

Comment:
Keep it in a different class. Have an enum and update graph search to include random walk